modules/__tests__/Link-test.js

```diff
@@ -557,4 +557,28 @@ describe('A <Link>', () => {
       })
     })
   })
+
+  describe('when the "noHref" prop is specified', function () {
+    class App extends Component {
+      render() {
+        return (
+          <div>
+            <Link noHref>Link with noHref</Link>
+          </div>
+        )
+      }
+    }
+
+    it('returns an anchor tag without an href', function (done) {
```

Comment:
Also test that clicking on it actually works as done [here](https://github.com/salemove/react-router/blob/3ecfd5ced597799578af001e9636a34a002feffb/modules/__tests__/Link-test.js#L365) for example.